packages/manager/src/managers/prismicRepository/types.ts

```diff
@@ -167,3 +167,32 @@ export const Environment = t.type({
 	),
 });
 export type Environment = t.TypeOf<typeof Environment>;
+
+export const OnboardingStepId = t.union([
+	t.literal("createPrismicProject"),
+	t.literal("chooseLocale"),
+	t.literal("createProject"),
+	t.literal("setupSliceMachine"),
+	t.literal("createPageType"),
+	t.literal("codePage"),
+	t.literal("createSlice"),
+	t.literal("reviewAndPush"),
+	t.literal("createContent"),
+	t.literal("renderPage"),
+]);
+export type OnboardingStepId = t.TypeOf<typeof OnboardingStepId>;
+
+export const OnboardingState = t.type({
+	completedSteps: t.array(OnboardingStepId),
+	isDismissed: t.boolean,
+	context: t.type({
+		framework: t.union([
+			t.literal("next"),
+			t.literal("nuxt"),
+			t.literal("sveltekit"),
+			t.literal("other"),
```

Comment:
❓ How can it be `other`?

Reply:
Same as my comment on editor PR, SM is something today only for the 3 frameworks above, we don't support anything else than that. So I don't understand all this other management.

Reply:
Honestly, here is doesn't really matter no? These are the possible values for this field, if it's `other`, the onboarding guide won't be show, but this can still return `other`.

Reply:
But for me it's like writing "toto" and saying it's not a problem. It's not, but why writing it? Why not removing it if it's not possible?

Reply:
Yeah sure, I don't have a strong opinion here, but we should still validate it somewhere no?

Removed this ✅

Reply:
Hum, it will never happen so it's good without so. I mean you know the code better. But for me if other never happen no need to add it.